src/d3a/models/strategy/home_meter.py

```diff
@@ -189,6 +190,21 @@ def event_market_cycle(self):
 
         self._delete_past_state()
 
+    def set_real_energy_of_last_market(self):
+        """Set the (simulated) actual energy of the device in the previous market slot."""
+        if self.area.current_market:
+            self._set_real_energy_kWh(self.area.current_market.time_slot)
+
+    def _set_real_energy_kWh(self, time_slot: DateTime) -> None:
+        """Set the (simulated) actual energy of the device in a market slot."""
+        energy_forecast_kWh = self.state.get_energy_at_market_slot(time_slot)
+        simulated_real_energy = utils.alter_energy(energy_forecast_kWh)
+
+        # This value can be either positive or negative. This is different than with the other
```

Comment:
Please add again the sign convention to this comment (which sign means consumption and which production), thanks!